packages/zoe/src/contracts/mintAndSellNFT.js

```diff
@@ -1,101 +1,117 @@
-/* eslint-disable no-use-before-define */
+// /* eslint-disable no-use-before-define */
 // @ts-check
 
 import harden from '@agoric/harden';
 import produceIssuer from '@agoric/ertp';
 
-// This contract mints non-fungible tokens and creates an instance of
-// a selling contract to sell the tokens in exchange for some sort of money.
+/**
+ * This contract mints non-fungible tokens and creates an instance of
+ * a selling contract to sell the tokens in exchange for some sort of money.
+ *
+ * makeInstance() returns an invitation that, when exercised, returns a
+ * ticketMaker with a .sellTokens() method. sellTokens() takes a specification
+ * of what is being sold
+ * {
+ *   customExtentProperties: { ...arbitrary },
+ *   count: 3,
+ *   moneyIssuer: moolaIssuer,
+ *   sellItemsInstallationHandle,
+ *   pricePerItem: moolaAmountMath.make(20),
+ * }
+ * and returns the payouts in `outcome`, and an API that allows selling the
```

Comment:
I think we should instead say that the 'outcome' is an offerResult or something similar. It sounds like the payouts are being returned directly, rather than the whole object that includes the payouts property.

Reply:
Is this what you meant? I'm not sure this is simple and straightforward yet.

Reply:
Sound good enough to me, thanks!